packages/hooks/useCurrentUser.js

```diff
@@ -9,5 +10,18 @@ export default function useCurrentUser (useQueryArgs = {}) {
     return null
   }
 
-  return [{ currentUser: data?.me, fetching, error, stale }, queryCurrentUser]
+  // Memoized validation of memberships to ensure group property is always an object
+  const validatedCurrentUser = useMemo(() => {
```

Comment:
do we need this still after the fix above? just adds confusion

Reply:
fair enough. i wouldnt really call this validation, more presentation. Also it would just be hiding other bugs again if the group was blank, but thats ok, better than a crash